Altis_Life.Altis/core/init.sqf

```diff
@@ -97,7 +97,7 @@ if (LIFE_SETTINGS(getNumber,"pump_service") isEqualTo 1) then {
 };
 
 life_fnc_RequestClientId = player;
-publicVariableServer "life_fnc_RequestClientId"; 
+publicVariableServer "life_fnc_RequestClientId";
```

Comment:
no clue, somehow travisci cancelled the check and i had to do a BS commit to fix it